data/timeline.service.in

```diff
@@ -5,5 +5,5 @@ Documentation=man:snapper(8) man:snapper-configs(5)
 
 [Service]
 Type=simple
-ExecStart=/usr/lib/snapper/systemd-helper --timeline
+ExecStart=@libexecdir@/snapper/systemd-helper --timeline
```

Comment:
This results in ExecStart=${exec_prefix}/libexec/snapper... for me. AFAIS that does not work, see e.g. https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=729982.